extensions/amp-form/0.1/form-proxy.js

```diff
@@ -76,26 +78,27 @@ function createFormProxyConstr(win) {
   prototypes.forEach(function(prototype) {
     const properties = win.Object.getOwnPropertyDescriptors(prototype);
     for (const name in properties) {
-      if (win.Object.hasOwnProperty(FormProxyProto, name) ||
-          name == 'constructor') {
+      if (win.Object.hasOwnProperty(FormProxyProto, name)) {
```

Comment:
Oh, and this should be `.call`, and really `Object.prototype.hasOwnProperty.call`. Or we created a [helper](https://github.com/ampproject/amphtml/blob/master/src/utils/object.js#L44-L46).